backend/src/src-licenseinfo/src/main/java/org/eclipse/sw360/licenseinfo/LicenseInfoHandler.java

```diff
@@ -257,4 +267,15 @@ protected LicenseInfoParsingResult noSourceParsingResult() {
         return outputGenerators.stream().filter(outputGenerator -> generatorClassname.equals(outputGenerator.getClass().getName()))
                 .findFirst().orElseThrow(() -> new TException("Unknown output generator: " + generatorClassname));
     }
+
+    private static String loadDefaultLicenseInfoHeaderText(){
+        try {
+            String defaultLicenseInfoHeader = IOUtils.toString(LicenseInfoHandler.class.getResourceAsStream(DEFAULT_LICENSE_INFO_HEADER_FILE), "UTF-8");
```

Comment:
You have not changed to use `CommonUtils.loadResource()` as requested. I still think it is rather important to do so.